packages/components/addon/components/hds/disclosure/index.js

```diff
@@ -48,12 +43,12 @@ export default class HdsDisclosureComponent extends Component {
 
   @action
   close() {
-    if (this.isOpen) {
+    schedule('afterRender', () => {
```

Comment:
@alex-ju missing a comment here why this is necessary (not sure why you have removed it)

Reply:
good spot, I'll add a new one